skptesting/benchmark-proxy.sh

```diff
@@ -1,12 +1,12 @@
 #! /bin/bash
 
+source "${GOPATH}/src/github.com/zalando/skipper/skptesting/benchmark.inc"
+
 if [ "$1" == -help ]; then
-	log benchmark-proxy.sh [duration] [connections] [warmup-duration]
+	log "benchmark-proxy.sh [duration] [connections] [warmup-duration] [skipper-binary]"
 	exit 0
 fi
 
-source $GOPATH/src/github.com/zalando/skipper/skptesting/benchmark.inc
```

Comment:
Is this moved due to missing log()?

The downside is that benchmark.inc now install skipper even if someone just ran it with -help

Reply:
Maybe just change it to `echo ""benchmark-proxy.sh [duration] ..." >&2`. I think it may also print to stdout since its not an error but help message

Reply:
yes it is required to use log()

Reply:
I guess it does not hurt to install a current local skipper